lemur/migrations/versions/1ae8e3104db8_.py

```diff
@@ -14,28 +14,8 @@
 
 
 def upgrade():
-    connection = None
-
-    if not op.get_context().as_sql:
-        connection = op.get_bind()
-        connection.execution_options(isolation_level='AUTOCOMMIT')
-
-    op.execute("ALTER TYPE log_type ADD VALUE 'create_cert'")
-    op.execute("ALTER TYPE log_type ADD VALUE 'update_cert'")
-
-    if connection is not None:
-        connection.execution_options(isolation_level='READ_COMMITTED')
+    op.sync_enum_values('public', 'log_type', ['key_view'], ['create_cert', 'key_view', 'update_cert'])
```

Comment:
Thanks for fixing this migration, I hit this issue myself as well and had to work around it.